src/build.py

```diff
@@ -755,6 +755,8 @@ def LLVM():
              '-DCOMPILER_RT_INCLUDE_TESTS=OFF',
              '-DLLVM_BUILD_LLVM_DYLIB=%s' % build_dylib,
              '-DLLVM_LINK_LLVM_DYLIB=%s' % build_dylib,
+             # Our mac bot's toolchain's ld64 is too old for trunk libLTO.
+             '-DLLVM_TOOL_LTO_BUILD=OFF',
```

Comment:
Right. libLTO (not to be confused with the LLVM gold plugin, which also does LTO) is really only used for mac's ld64 AFAIK, even though you can also build it on Linux. And since we aren't using this libLTO for wasm, it's find to disable for now.